File: TrainingZ.Domain/Models/UserInfo.cs
namespace TrainingZ.Domain.Models;

public class UserInfo(string goals, string sleepDiet, string activity, string injuries, string timeAvaiable, string other)
{
    public string Goals { get; set; } = goals;
    public string SleepDiet { get; set; } = sleepDiet;
    public string Activity { get; set; } = activity;
    public string Injuries { get; set; } = injuries;
    public string TimeAvaiable { get; set; } = timeAvaiable;
    public string Other { get; set; } = other;
}

File: training-z/src/app/modules/dashboard/models/user-info.ts
export type UserInfo = {
  goals: string;
  sleepDiet: string;
  activity: string;
  injuries: string;
  timeAvaiable: string;
  other: string;
};

export const QuestionKeys: (keyof UserInfo)[] = [
  'goals',
  'sleepDiet',
  'activity',
  'injuries',
  'timeAvaiable',
  'other',
];
